--- src/app/spartacus/spartacus-configuration.module.ts
+++ src/app/spartacus/spartacus-configuration.module.ts
@@ -13,7 +13,13 @@ import { defaultCmsContentProviders, layoutConfig, mediaConfig } from "@spartacu
       }
     },
   }), provideConfig(<SiteContextConfig>{
-    context: {},
+    context: {
+      // TODO: adjust parameters and site
+      urlParameters: ['baseSite', 'language', 'currency'],
+      baseSite: ['electronics-spa'],
+      currency: ['USD'],
+      language: ['en'],
+    },
   }), provideConfig(<I18nConfig>{
     i18n: {
       resources: { en: translationsEn },